pettingzoo/utils/wrappers/base_parallel.py

```diff
@@ -17,37 +22,47 @@ def __init__(self, env):
 
         # Not every environment has the .state_space attribute implemented
         try:
-            self.state_space = self.env.state_space
+            self.state_space = self.env.state_space  # type: ignore
```

Comment:
Is this the best way? I think pyright ignore is better as it specifies which check it is for